### PR TITLE
Updates README.md to include instructions for launching Triton

### LINKED ISSUE #288: [DOC] README.md should include instructions on how to launch Triton
## Report incorrect documentation
Most of our models/examples are intended to work with Triton.

**Location of incorrect documentation**
`README.md`

### PR DESCRIPTION
**Pull request by dagardner-nv**
Also updates:
* Git LFS instructions
* Software requirements
* Specifics that apply to all users were moved from `CONTRIBUTING.md` to `README.md`
* Default version of Triton used by Validation scripts updated to 22.06 

Fixes #288

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -179,7 +150,7 @@ Note: These instructions assume the user is using `mamba` instead of `conda` sin
 
 - Pascal architecture or better
```
Comment (pdmack) [inline CONTRIBUTING.md:151]: "Pascal architecture GPU or better"
or
"GPU with Compute Capability of 6.0 or higher"

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -179,7 +150,7 @@ Note: These instructions assume the user is using `mamba` instead of `conda` sin
...
 - NVIDIA driver `450.80.02` or higher
-- [CUDA 11.0+](https://developer.nvidia.com/cuda-downloads)
+- [Cuda 11.5](https://developer.nvidia.com/cuda-11-5-2-download-archive)
```
Comment (pdmack) [inline CONTRIBUTING.md:153]: nit "CUDA"

File: README.md [hunk 3]
```diff
@@ -76,6 +96,22 @@ DOCKER_IMAGE_TAG="v22.06.00a-runtime" ./docker/run_container_release.sh
...
 
+## Launching Triton Server
+
+Many of the validation tests and example workflows require a Triton server to function.
+Use the following command to launch a Docker container for Triton loading all of the included pre-trained models:
+
+```bash
+docker run --rm -ti --gpus=all -p8000:8000 -p8001:8001 -p8002:8002 \
+	-v $PWD/models:/models \
+	nvcr.io/nvidia/tritonserver:22.02-py3 \
```
Comment (pdmack) [inline README.md:107]: same as above, "22.06"

File: README.md [hunk 4]
```diff
@@ -21,33 +21,53 @@ More advanced users, or those who are interested in using the latest pre-release
 The following sections must be followed prior to building the Morpheus container or building Morpheus from source.
 
 #### Requirements
-- Pascal architecture or better
+- Pascal architecture GPU or better
 - NVIDIA driver `450.80.02` or higher
 - [Docker](https://docs.docker.com/get-docker/)
 - [The NVIDIA container toolkit](https://docs.nvidia.com/datacenter/cloud-native/container-toolkit/install-guide.html#docker)
+- [NVIDIA Triton Inference Server](https://catalog.ngc.nvidia.com/orgs/nvidia/containers/tritonserver) `22.02` or higher
```
Comment (pdmack) [inline README.md:28]: Based on #295 , this should probably be 22.06 now due to the somewhat strict compatibility requirements for Treelite in the ransomware models

**Review by pdmack — CHANGES_REQUESTED** [review]

**Review by mdemoret-nv — APPROVED** [review]
LGTM

**Review by pdmack — APPROVED** [review]
LGTM

**Comment by dagardner-nv** [comment]
@gpuci merge

**Comment by dagardner-nv** [comment]
@gpucibot merge

**Pull request merged**